RELEASE-NOTES.txt

```diff
@@ -2,6 +2,8 @@
 
 17.9
 -----
+* [***] Block editor: New Block: Embed block. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3727]
+* [*] Enables Support for Global Style Colors with Full Site Editing Themes [https://github.com/wordpress-mobile/WordPress-Android/pull/15002]
```

Comment:
We should probably remove this entry as [this PR](https://github.com/wordpress-mobile/WordPress-Android/pull/15058) disabled it.

Reply:
Done in https://github.com/wordpress-mobile/WordPress-Android/pull/15054/commits/a8acc247e98c0dc7ca656ce3a6174908980e3b84.

Reply:
Thanks 🙇 !